stories/1-base/typography.stories.js

```diff
@@ -7,20 +7,20 @@ export default {
 export const headings = () => (
   <>
     <div>
-      <h1>heading 1</h1>
-      <h2>heading 2</h2>
-      <h3>heading 3</h3>
-      <h4 className="font-united">heading 4</h4>
-      <h5 className="font-united">heading 5</h5>
-      <h6 className="font-united">heading 6</h6>
-    </div>
-    <div className="text-right">
       <h1 className="font-gotham">heading 1</h1>
```

Comment:
@marinagustint Any reason why the order of fonts being displayed was changed? We should have united on the left, then gotham on the right since united is the primary font for headings.